Source/Core/Common/Common.vcxproj.filters

```diff
@@ -189,6 +189,8 @@
     <ClInclude Include="GL\GLInterfaceBase.h">
       <Filter>GL\GLInterface</Filter>
     </ClInclude>
+    <ClInclude Include="Assert.h" />
```

Comment:
No, visual studio automatically added them on save.

I see no reason to put them in a separate PR.